### PR TITLE
add travis badge, sweep readme

### PR DESCRIPTION
**Pull request by ngokevin**
- github badge
- more conciseness
- defer to aframe.io more, focus readme for devs and passerbys

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,34 @@
...
-<a-scene>
-    <a-entity camera="fov: 45"></a-entity>
-    <a-entity position="0 0 -10" rotation="45 30 0"
-              geometry="primitive: box; height: 4; width: 4; depth: 4"
-              material="color: green"></a-entity>
-</a-scene>
-```
+- [Guide](https://aframe.io/docs/guide/)
+- [Docs](https://aframe.io/docs/core/)
+- [Examples](https://aframe.io)
```
Comment (cvan) [inline README.md]: https://aframe.io/examples/

File: README.md [hunk 2]
```diff
@@ -68,20 +40,19 @@ To start the local development server:
...
-
 ### Running tests
 
-After you have [cloned the repo and installed the dependencies](#local-installation-and-development), simply run the tests like so:
+After [cloning and installing](#local-installation-and-development), to run tests:
 
     npm test
 
-## Browser console logging
+## Browser Console Logging
```
Comment (cvan) [inline README.md]: capitalise "tests" above then

File: README.md [hunk 3]
```diff
@@ -1,62 +1,34 @@
...
 
-> This library is specific to the core A-Frame library. For more information about the A-Frame project as a whole, head over to [the A-Frame site](https://aframe.io) or [the `aframe` repo](https://github.com/aframevr/aframe/).
+> There are currently plans for this repository to be the canonical [`aframe`](https://github.com/aframevr/aframe).
 
-## Entity-Component System
+A-Frame is a library for creating virtual reality web experiences.
 
-[Entity-Component System](https://en.wikipedia.org/wiki/Entity_component_system) is an architectural pattern commonly used in game engines such as Unity, PlayCanvas, an Unreal Engine 4+.
+- **Virtual Reality**: Drop in the library and have a virtual reality scene within a few lines of markup.
+- **Based on the DOM**: Manipulate with Javascript, use with your favorite libraries and frameworks.
```
Comment (cvan) [inline README.md]: "Javascript" -> "JavaScript"

Comment (cvan) [inline README.md]: for "libraries and frameworks" perhaps link to https://github.com/aframevr/awesome-aframe/ ?

File: README.md [hunk 4]
```diff
@@ -1,62 +1,34 @@
...
+![travis](https://travis-ci.org/aframevr/aframe-core.svg)
 
-> This library is specific to the core A-Frame library. For more information about the A-Frame project as a whole, head over to [the A-Frame site](https://aframe.io) or [the `aframe` repo](https://github.com/aframevr/aframe/).
+> There are currently plans for this repository to be the canonical [`aframe`](https://github.com/aframevr/aframe).
 
-## Entity-Component System
+A-Frame is a library for creating virtual reality web experiences.
 
-[Entity-Component System](https://en.wikipedia.org/wiki/Entity_component_system) is an architectural pattern commonly used in game engines such as Unity, PlayCanvas, an Unreal Engine 4+.
+- **Virtual Reality**: Drop in the library and have a virtual reality scene within a few lines of markup.
```
Comment (cvan) [inline README.md]: namedrop WebVR :microphone: ?

**Comment by cvan** [comment]
I like all of these changes. thanks a ton!

now [aframe](https://github.com/aframevr/aframe/blob/master/README.md) next :smirk:

**Comment by ngokevin** [comment]
thanks!

**Pull request merged**